pkg/api/validation/validation.go

```diff
@@ -2773,11 +2773,11 @@ func ValidateResourceRequirements(requirements *api.ResourceRequirements, fldPat
 		// Check that request <= limit.
 		requestQuantity, exists := requirements.Requests[resourceName]
 		if exists {
-			// For GPUs, require that no request be set.
-			if resourceName == api.ResourceNvidiaGPU {
-				allErrs = append(allErrs, field.Invalid(reqPath, requestQuantity.String(), "cannot be set"))
+			// For GPUs, require that `requests` never be less than `limits`.
```

Comment:
Well, for all resources it's already the case that a request can't exceed the limit. In this case, we _also_ require for it to never be less than the limit, either. Perhaps I should reword the comment to make that clearer.